core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -578,6 +588,9 @@ private String toStringSlow() {
         // Build the string representation.
         final StringBuilder buf = TemporaryThreadLocals.get().stringBuilder();
         buf.append("[creqId=").append(creqId);
+        if (parent != null) {
+            buf.append(", pcreqId=").append(pcreqId);
+        }
         if (sreqId != null) {
             buf.append(", sreqId=").append(sreqId);
         }
```

Comment:
How about just `preqId`?

Reply:
I meant renaming `pcreqId` to `preqId`, just in case it wasn't clear.

Reply:
I thought the `preqId` too. `preqId` sounds good to me. Let's go with `preqId`.